packages/nouns-webapp/src/pages/Explore/index.tsx

```diff
@@ -0,0 +1,250 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { BigNumber } from 'ethers';
+import classes from './Explore.module.css';
+import cx from 'classnames';
+import ExploreNounDetail from '../../components/ExploreGrid/ExploreNounDetail';
+import { motion, AnimatePresence } from 'framer-motion/dist/framer-motion';
+import { Auction as IAuction } from '../../wrappers/nounsAuction';
+import { useAppSelector } from '../../hooks';
+import dotenv from 'dotenv';
+import { useKeyPress } from '../../hooks/useKeyPress';
+import ExploreNav from '../../components/ExploreGrid/ExploreNav';
+import ExploreGrid from '../../components/ExploreGrid';
+dotenv.config();
```

Comment:
Already run in `index.ts`

Reply:
resolved in https://github.com/nounsDAO/nouns-monorepo/pull/595/commits/ac419510caf31635bd6269e46d86cdd58e0a7397